frontend/src/utils/api.js

```diff
@@ -9,6 +9,8 @@ const API_URL =
     : 'http://localhost:8080' // make sure your backend is running on this port.
 // if your frontend can't connect, try the normal IP
 
+const AUTH_API_URL = 'https://h4i-infra-server.n3a9.now.sh'
```

Comment:
This shouldn't be in the code base, maybe `.env` or something.

Reply:
Maybe this is fine, depends on what someone could do with the link.

Reply:
I'm not sure if this counts as an environment variable. I'm pretty sure it has the same amount of functionality as the other API URL in this file, but I can move both of them to the .env file if you want

Reply:
Nvm, I think this is fine for now. I just need a quick readme on the setup process for auth for this project specifically, and then it should be good to go from my end. I'll just setup the auth server from my end and switch in the URL.

Reply:
Change to `'h4i-portal-infra-server.now.sh'`